src/test/java/Homework23.java

```diff
@@ -0,0 +1,23 @@
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework23 extends BaseTest{
+
+    //Prerequisite - empty playlist to delete
+
+    @Test
+    public void deletePlaylist() throws InterruptedException{
+        String expectedPLayListDeletedMessage = "Deleted playlist \"Golden Girls.\"";
+
+        provideEmail("adam.johnson@testpro.io");
+        providePassword("1Te$t$tudent");
+        clickSubmit();
+        openPlaylist();
+        clickDeletePlaylistBtn();
+        Assert.assertEquals(getDeletedPlaylistMsg(), expectedPLayListDeletedMessage);
+    }
```

Comment:
**Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
   
   For example
   
   
![Clean_Test](https://github.com/Test-Pro-Bootcamp/A51-QA-Automation-Framework/assets/104761482/a282b186-cc4f-4c5d-a61f-df14f94611a7)